### ISSUE TITLE
Eval bug: W load: control-looking token: 128247 '</s>' was not control-type;

### ISSUE THREAD

**Issue by italkido** [ggml-org/llama.cpp#23183]
### Name and Version

version: 9190 (b64739ea3)
built with GNU 11.4.0 for Linux x86_64

env: ubuntu 26.4 + rocm 7.2.3 
llm model :  qwen3-coder-next

W load: control-looking token: 128247 '</s>' was not control-type; this is probably a bug in the model. its type will be overridden

### Operating systems

Linux

### GGML backends

HIP

### Hardware

Name:                    AMD RYZEN AI MAX+ 395 w/ Radeon 8060S
  Uuid:                    CPU-XX                             
  Marketing Name:          AMD RYZEN AI MAX+ 395 w/ Radeon 8060S
  Vendor Name:             CPU                                
----------------------------------------------------------------
Agent 2                  
*******                  
  Name:                    gfx1151                            
  Uuid:                    GPU-XX                             
  Marketing Name:          AMD Radeon Graphics                
  Vendor Name:             AMD                                
  Feature:                 KERNEL_DISPATCH                    
  Profile:                 BASE_PROFILE                       
  Float Round Mode:        NEAR                               
  Max Queue Number:        128(0x80)                          
  Queue Min Size:          64(0x40)                           
  Queue Max Size:          131072(0x20000)                    
  Queue Type:              MULTI                              
  Node:                    1                                  
  Device Type:             GPU                    

### Models

Qwen-coder-next

### Problem description & steps to reproduce

llama-server -m /models/unsloth/coder/q8/Qwen3-Coder-Next-Q8_0-00001-of-00003.gguf --host 0.0.0.0 --port 8666 -ngl 99 --no-mmap -fa on

### Relevant log output

<details>
<summary>Logs</summary>

```console

```
</details>

**Comment by alan-codes-things** [comment]
I got the same warning for unsloth Qwen3 Coder Next.

```
...
0.00.675.224 I srv    load_model: loading model '/home/dev/.cache/huggingface/hub/models--unsloth--Qwen3-Coder-Next-GGUF/snapshots/ce09c67b53bc8739eef83fe67b2f5d293c270632/Qwen3-Coder-Next-UD-Q4_K_M.gguf'
0.00.675.237 I common_init_result: fitting params to device memory ...
0.00.675.238 I common_init_result: (for bugs during this step try to reproduce them with -fit off, or provide --verbose logs if the bug only occurs with -fit on)
0.01.714.061 W load: control-looking token: 128247 '</s>' was not control-type; this is probably a bug in the model. its type will be overridden.
...
```
`
./llama-server -hf unsloth/Qwen3-Coder-Next-GGUF:UD-Q4_K_M 
`
It's probably the model not llama.cpp.

Model output looks normal. Probably doesn't effect logits.

**Comment by JEAYNE** [comment]
Same Warning here
 - Windows 11;  RTX 5090 (32606 MiB, 30991 MiB free);   CUDA 13.3
 - llama.cpp 9756 (d0f9d2e5a) 
 - qwen3-coder-30b-a3b-instruct-q8_0.gguf

```
llama-server.exe --fim-qwen-30b-default --no-mmap
...
0.00.560.921 I srv    load_model: loading model 'C:\Users\XXXXX\.cache\huggingface\hub\models--ggml-org--Qwen3-Coder-30B-A3B-Instruct-Q8_0-GGUF\snapshots\d2a13024f16ef1985cb8347c3dbad6114e0ecef6\qwen3-coder-30b-a3b-instruct-q8_0.gguf'
0.00.560.931 I common_init_result: fitting params to device memory ...
0.00.560.931 I common_init_result: (for bugs during this step try to reproduce them with -fit off, or provide --verbose logs if the bug only occurs with -fit on)
0.02.433.577 W load: control-looking token: 128247 '</s>' was not control-type; this is probably a bug in the model. its type will be overridden
```

**Comment by fatrmiloon-hub** [comment]
I can confirm the same warning as well. It seems to occur across different
platforms and model variants, which suggests it may be model-related rather
than backend-specific.
